pkg/rds.go

```diff
@@ -515,6 +540,22 @@ func (e *RDSExporter) addAllInstanceMetrics(sessionIndex int, instances []*rds.D
 			e.cache.AddMetric(prometheus.MustNewConstMetric(MaxConnectionsMappingError, prometheus.GaugeValue, 1, e.getRegion(sessionIndex), *instance.DBInstanceIdentifier, *instance.DBInstanceClass))
 		}
 
+		//Gets EOL for engine and version
+		if eolInfo, ok := eolMap[*instance.Engine][*instance.EngineVersion]; ok {
+			level.Info(e.logger).Log("msg", fmt.Sprintf("EOL for Engine %s, Version %s: %s\n", *instance.Engine, *instance.EngineVersion, eolInfo.EOL))
+			e.cache.AddMetric(prometheus.MustNewConstMetric(EOLDate, prometheus.GaugeValue, 1, e.getRegion(sessionIndex), *instance.DBInstanceIdentifier, *instance.Engine, *instance.EngineVersion, eolInfo.EOL))
+			eolStatus, err := getEOLStatus(eolInfo.EOL)
+			if err != nil {
+				level.Info(e.logger).Log("msg", fmt.Sprintf("Could not get days to EOL for Engine %s, Version %s\n", *instance.Engine, *instance.EngineVersion))
+				//Add empty eol status?
+			}
```

Comment:
The only case for err is `EOL` string parse error, better to log an `Error` level log with `err` object.

Reply:
I mean include `err` message in the log so we can see the raw error.

```golang
level.Error(e.logger).("msg", ..., "error", err.Error())
```

btw, when it's `err`, `eolStatus` will be empty string, do you still want to log a info log and `AddMetric`? Otherwise put those logic in a `else` branch, may worth adding a test case for error scenario.